### ISSUE TITLE
Test failures when another malfunctioning solver is available and overrides SDPT3

### ISSUE THREAD

**Issue by denisrosset** [replab/replab#127]
Observed today on a computer with a MOSEK installation and an expired license.

All SDP related tests, but the test failures are not informative.

```
--------------------------------------------------

failure: Error using CommutantVarTest>test_fromSdpMatrix_SDP_CHSH (line 78)
Symmetrized SDP doesn't yield the same result as the non-symmetrized one
  test_fromSdpMatrix_SDP_CHSH:78 (C:\Users\emman\OneDrive\MATLAB\replab-0.5.0\tests\CommutantVar\CommutantVarTest.m)

failure: Error using CommutantVarTest>test_fromSdpMatrix_SDP_CHSH_FullProb (line 141)
Symmetrized SDP doesn't yield the same result as the non-symmetrized one
  test_fromSdpMatrix_SDP_CHSH_FullProb:141 (C:\Users\emman\OneDrive\MATLAB\replab-0.5.0\tests\CommutantVar\CommutantVarTest.m)

failure: Error using CommutantVarTest>test_fromSdpMatrix_SDP_CGLMP3_FullProb (line 228)
Symmetrized SDP doesn't yield the same result as the non-symmetrized one
  test_fromSdpMatrix_SDP_CGLMP3_FullProb:228 (C:\Users\emman\OneDrive\MATLAB\replab-0.5.0\tests\CommutantVar\CommutantVarTest.m)

failure: Error using CommutantVarTest>test_fromSymSdpMatrix (line 264)
Block-diagonalized SDP doesn't yield the same result as the non-block-diagonalized one
  test_fromSymSdpMatrix:264 (C:\Users\emman\OneDrive\MATLAB\replab-0.5.0\tests\CommutantVar\CommutantVarTest.m)

failure: Error using CommutantVarTest>test_fromIndexMatrix (line 301)
Block-diagonalized SDP doesn't yield the same result as the non-block-diagonalized one
  test_fromIndexMatrix:301 (C:\Users\emman\OneDrive\MATLAB\replab-0.5.0\tests\CommutantVar\CommutantVarTest.m)

failure: Error using eqTest>test_double_combination (line 57)
Assertion failed.
  test_double_combination:57 (C:\Users\emman\OneDrive\MATLAB\replab-0.5.0\tests\CommutantVar\eqTest.m)

failure: Error using SedumiDataTest>test_CHSH (line 18)
Assertion failed.
  test_CHSH:18 (C:\Users\emman\OneDrive\MATLAB\replab-0.5.0\tests\SedumiDataTest.m)

--------------------------------------------------

FAILED (passed=747, failure=7)
```

**Comment by denisrosset** [comment]
I see several ways out of that.

1. Force the RepLAB tests to use SDPT3 by the relevant `sdpsettings` option

2. Try to solve a simple SDP as part of the detection of solvers in `replab_addpaths`

**Comment by jdbancal** [comment]
Good catch, let's always run an SDP in replab_addpaths, this would be a good check to do anyway!

**Comment by jdbancal** [comment]
Fixed by #128
